goenvtemplator.go

```diff
@@ -105,8 +189,8 @@ func main() {
 		log.Print("Generating templates")
 	}
 
-	if err := generateTemplates(tmpls, debugTemplates, delimLeft, delimRight); err != nil {
-		log.Fatal(err)
+	if err := generateTemplates(tmpls, delimLeft, delimRight, engine); err != nil {
+		panic(err)
```

Comment:
Can you please explain the reason of the change?

Reply:
I think it is better to panic than proceed to execute a command with partially generated templates.

Reply:
https://golang.org/pkg/log/#Fatal

Fatal calls `os.Exis` so it will not proceed.

Reply:
Oh, didn't know that. Will fix.